tests/publiccloud/aistack_rbac_run.pm

```diff
@@ -0,0 +1,70 @@
+use Mojo::Base 'publiccloud::basetest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+use publiccloud::utils;
+use version_utils;
+
+sub run_python_script {
+    my $script = shift;
+    my $logfile = "output.txt";
+    record_info($script, "Running python script: $script");
+    assert_script_run("$script");
+    assert_script_run("./$script 2>&1 | tee $logfile");
+}
+
+sub run {
+    my ($self, $args) = @_;
+
+    my $instance = $self->{my_instance} = $args->{my_instance};
+    my $provider = $self->{provider} = $args->{my_provider};
+
+    # Get Open WebUI ip address, add it to /etc/hosts, and verify connectivity
+    my $ipaddr = get_var('OPENWEBUI_IP');
+    my $host_name = get_var('OPENWEBUI_HOSTNAME');
+    record_info("debug $ipaddr");
```

Comment:
you can see values of all variables in settings tab not sure why we need this line